src/main/java/me/totalfreedom/totalfreedommod/discord/DiscordToAdminChatListener.java

```diff
@@ -86,8 +86,12 @@ public void onMessageReceived(MessageReceivedEvent event)
                 {
                     for (Message.Attachment attachment : msg.getAttachments())
                     {
-                        TextComponent text = new TextComponent(mediamessage);
+                        TextComponent text = new TextComponent((msg.getContentDisplay().isEmpty() ? "" : " ") + mediamessage);
```

Comment:
Not needed as there is a check on line 91.

Reply:
oops, fixed